src/ontology/cl-edit.owl

```diff
@@ -26612,6 +26612,8 @@ EquivalentClasses(obo:CL_1000742 ObjectIntersectionOf(obo:CL_1000612 ObjectSomeV
 SubClassOf(obo:CL_1000742 obo:CL_0000650)
 SubClassOf(Annotation(oboInOwl:is_inferred "true") obo:CL_1000742 obo:CL_1000612)
 SubClassOf(obo:CL_1000742 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0002320))
+SubClassOf(Annotation(oboInOwl:hasDbXref "PMID:9127003"^^xsd:string) obo:CL_1000742 ObjectSomeValuesFrom(obo:RO_0002215 obo:GO_0006909))
+SubClassOf(Annotation(oboInOwl:hasDbXref "https://doi.org/10.1152/physrev.1998.78.3.723"^^xsd:string) obo:CL_1000742 ObjectSomeValuesFrom(obo:RO_0002215 obo:GO_0070252))
 
```

Comment:
Adding refs on subClassOf axioms is good for traceability but I don't think it standard practice at present (@shawntanzk  - are we doing this elsewhere?)

Reply:
Yes, we do it in BDSO for expresses axioms. I think we might also add refs to some taxon annotations which is basically adding it to subclass axioms in a roundabout way. That being said, if the subclass axioms matches the definitions, is there a reason for adding it on the axiom rather than just the definition?

Reply:
I think you might want to add a comment explaining this axioms "[glomerular mesangial cell](http://purl.obolibrary.org/obo/CL_1000742) SubClassOf [capable of](http://purl.obolibrary.org/obo/RO_0002215) some [actin-mediated cell contraction](http://purl.obolibrary.org/obo/GO_0070252)" and add the dbxref there instead